Source/ablastr/fields/PoissonSolver.H

```diff
@@ -125,7 +126,7 @@ computePhi (amrex::Vector<std::unique_ptr<amrex::MultiFab> > const & rho,
     // scale rho appropriately; also determine if rho is zero everywhere
     amrex::Real max_norm_b = 0.0;
     for (int lev=0; lev<=finest_level; lev++) {
-        rho[lev]->mult(-1._rt/PhysConst::ep0);
+        rho[lev]->mult(-1._rt/PhysConst::ep0); // TODO: when do we "un-multiply" this? We need to document this side-effect!
```

Comment:
It is assumed that rho won't be used after phi is computed. Though I agree that this should be documented, perhaps in the documentation for the routine. Make rho [in/out] with an explanation. In WarpX, this assumption is correct, rho is not used afterward. For example, the rho diagnostics regenerate rho.